### PR TITLE
updated request to pull FreeBSD11enhanced client role

### PR DESCRIPTION
**Pull request by planet-winter**
Hi @pablodav 
created this fresh pull request as you suggested. 
can you please point me to the "notes sent" to keep sane defaults as of your mail. I don't know which notes you mean. 
Thanks
  Daniel

### CODE REVIEW

**Comment by pablodav** [comment]
I have noticed some lines that needs to be fixed, so I have prepared #13 issues to continue this discussion and will merge this into branch feature/13_freebsd_port.
Then will implement the fixes on that branch and we will discuss about the rest of the changes before merging to master.

Sane defaults just means defaults that doesn't breaks things and are optimal.

File: defaults/main.yml [hunk 1]
```diff
@@ -13,11 +18,11 @@ burp_client_pidfile: "/var/run/burp.pid"
 burp_client_password: "password"
 burp_client_protocol: "1"
 burp_client_syslog: "0"
-burp_client_ca_burp_ca: "/usr/sbin/burp_ca"
-burp_client_ca_csr_dir: "/etc/burp/CA-client"
-burp_client_ssl_cert_ca: "/etc/burp/ssl_cert_ca.pem"
-burp_client_ssl_cert: "/etc/burp/ssl_cert-client.pem"
-burp_client_ssl_key: "/etc/burp/ssl_cert-client.key"
+burp_client_ca_burp_ca: "{{ etc_dir }}/sbin/burp_ca"
```
Comment (pablodav) [inline defaults/main.yml:21]: This looks wrong, correct path is in /usr/sbin/burp_ca I think, will confirm and fix on #13 if necessary.

File: tasks/build-burp.yml [hunk 2]
```diff
@@ -11,7 +11,16 @@
   package:
     name: "{{ build_dependencies | join(',') }}"
     state: present
+  when: ( ansible_os_family != 'FreeBSD' and not burp_freebsd_use_portinstall )
```
Comment (pablodav) [inline tasks/build-burp.yml:14]: This conditional is breaking linux builds, as you can see here:
https://travis-ci.org/CoffeeITWorks/ansible_burp2_client/builds/340463642#L4015 

will merge into branch feature/13_freebsd_port and fix for #13

File: tasks/build-burp.yml [hunk 3]
```diff
@@ -11,7 +11,16 @@
...
 
+- name: install build dependencies
```
Comment (pablodav) [inline tasks/build-burp.yml:16]: will review for #13  as looks duplicated on the first view

File: tasks/freebsd.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+
+- name: install build dependencies
```
Comment (pablodav) [inline tasks/freebsd.yml:3]: looks like name is duplicated but task is different, could be good to change name. will check on #13

File: vars/FreeBSD-11.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  - automake
+  - uthash
+  - librsync2
+    
+# burp2_client_packages:
+  #  - burp
+
+# custom FreeBSD paths differet to the defaults
+burp_client_ca_burp_ca: "/usr/local/sbin/burp_ca"
+burp_client_ca_csr_dir: "{{ etc_dir }}/burp/CA-client"
```
Comment (pablodav) [inline vars/FreeBSD-11.yml:20]: all these etc_dir lines looks same as linux paths, will check on #13

File: defaults/main.yml (hunk 1 above)
Comment (planet-winter) [reply defaults/main.yml:21]: this one is definitely as you suggested /usr or /usr/local. I accidentally changed this in my last commit.

**Pull request merged**